### ISSUE TITLE
Issues with synergy after screens have locked (server and client)

### ISSUE THREAD

**Issue by dragon788** [synergy/synergy#4452]
Using Windows 7 Enterprise, I've enabled the "Secure Attention Sequence" via local Group Policy settings on the client. This works from a fresh boot (every time), or if I have locked the machine (sometimes requires restarting Synergy on the server), but will consistently fail to work if I have slept the client machine.

I was curious whether this is related to the fact that the IP address likely changed or was given away while the machine slept, but curiously, according to the Synergy console output, it is finding and moving the cursor over to my client machine, even though the cursor doesn't appear on the screen and keypresses will not unlock it. Simply unlocking the client machine manually allows synergy to resume working, but as this is a laptop closed on a docking station, it is not an optimal solution.

Probably unrelated:
On the server, if I have locked the screen and want to access the client, I can't do so from the Ctrl+Alt+Del screen, but once I have hit those keys and am at the user login/unlock page, I can move over to the client (I will test if adding the Secure Attention Sequence GP local fixes this on the server side).

**Comment by nbolton** [comment]
I renamed the title, as the old one didn't seem to be very specific. Is it still relevant?

**Comment by dragon788** [comment]
Yup, definitely relevant and the new title is probably more accurate.

**Comment by nbolton** [comment]
@dragon788 Thanks for confirming
